src/ui/server/helpers/has-form-data/index.ts

```diff
@@ -0,0 +1,53 @@
+import { RequestBody } from '../../../types';
+
+/**
+ * stripEmptyFormFields
+ * Filters out form fields with empty values
+ * @param {Express.Request.body} Form data
+ * @returns {Object} Form data without empty values
+ */
+export const stripEmptyFormFields = (formBody: RequestBody) => {
+  if (formBody) {
+    const fieldsWithValues = {};
+
+    const keys = Object.keys(formBody);
+
+    keys.forEach((key) => {
+      const value = formBody[key];
+
+      if (value !== '' && value !== null && value !== undefined) {
+        fieldsWithValues[key] = value;
+      }
+    });
+
+    return fieldsWithValues;
+  }
+
+  return {};
+};
+
+/**
+ * hasFormData
+ * Checks if form data has values
+ * @param {Express.Request.body} Form data
+ * @returns {Boolean}
+ */
+const hasFormData = (formBody: RequestBody) => {
+  if (!formBody) {
+    return false;
+  }
+
+  const { _csrf, ...formData } = formBody;
+
```

Comment:
A check to ensure `formData` is not empty unless this will never be a case?

Reply:
Already have a check for this just above https://github.com/UK-Export-Finance/exip/pull/176/files/197ca2b410b89bcbcbd329b3da63db4485e10663#diff-9385d5383b0ef70440e9453d1771ee81991bc905c6b21b77c85a006e04dbf131R36